src/plugins/mpath.c

```diff
@@ -110,7 +138,7 @@ gboolean bd_mpath_init () {
  *
  */
 void bd_mpath_close () {
-    /* nothing to do here */
+      /* nothing to do here */
```

Comment:
Why this change?

Reply:
Just a trivial miss type. Will remove it.